spec/person_spec.rb

```diff
@@ -0,0 +1,71 @@
+require_relative '../modules/person'
+
+describe Person do
+  let(:person) { Person.new(20) }
+  let(:book) { double('book') }
+
+  describe '#initialize' do
+    it 'sets the age attribute' do
+      expect(person.age).to eq(20)
+    end
+
+    it 'sets the id attribute with a unique identifier' do
+      expect(person.id).not_to be_nil
+    end
+
+    it 'sets the name attribute with a default value' do
+      expect(person.name).to eq('Unknown')
+    end
+
+    it 'sets the parent_permission attribute with a default value' do
+      expect(person.parent_permission).to be true
+    end
+
+    it 'initializes the rentals attribute as an empty array' do
+      expect(person.rentals).to be_empty
+    end
+  end
+
+  describe '#to_json' do
+    it 'returns a JSON string representation of the Person object' do
+      json = {
+        JSON.create_id => 'Person',
+        'id' => person.id,
+        'name' => 'Unknown',
+        'age' => 20,
+        'parent_permission' => true
+      }.to_json
+      expect(person.to_json).to eq(json)
+    end
+  end
+
+  describe '#can_use_services?' do
+    context 'when the person is 18 years or older' do
+      it 'returns true' do
+        expect(person.can_use_services?).to be true
+      end
+    end
+
+    context 'when the person is under 18 years old and has parental permission' do
+      let(:person) { Person.new(16, parent_permission: true) }
+
+      it 'returns true' do
+        expect(person.can_use_services?).to be true
+      end
+    end
+
+    context 'when the person is under 18 years old and does not have parental permission' do
+      let(:person) { Person.new(16, parent_permission: false) }
+
+      it 'returns false' do
+        expect(person.can_use_services?).to be false
+      end
+    end
+  end
+
+  describe '#correct_name' do
+    it 'returns the name attribute' do
+      expect(person.correct_name).to eq('Unknown')
+    end
```

Comment:
- [ ] Great on working on the changes that requested from the previous reviewer but I think you miss understand what he asked you, in Person you missed to add tests to the methods that he mentioned `of_age?, correct_name and add_rental` so great you added for `correct_name` but you still missed to test the another two methods and he said that (at least) an 'it' block 👇🏻  but it block for each method so at least one test for each method not at least one method 🙏🏻✅💯🤩 so kindly add tests for the another methods  `of_age? and add_rental` 💪🏻